src/main/java/com/example/userManagement/controller/UserController.java

```diff
@@ -0,0 +1,155 @@
+        return userServiceES.findUserByQuery(query);
+    }
+
+
+    /**
+     * GET /user/{userId} : Get user by user id
+     *
+     * @param userId (required)
+     * @return This is a user info!. (status code 200)
+     * or Error Occurred (status code 400)
+     * or Error Occurred (status code 404)
+     */
+    @ApiOperation(value = "Get user by user id", nickname = "getUserById", notes = "", response = User.class, tags = {"user operations",})
+    @ApiResponses(value = {
+            @ApiResponse(code = 200, message = "This is a user info!.", response = User.class),
+            @ApiResponse(code = 400, message = "Error Occurred", response = GetErrorResponse.class),
+            @ApiResponse(code = 404, message = "Error Occurred", response = GetErrorResponse.class)})
+    @GetMapping(
+            value = "/user/{userId}",
+            produces = {"application/json"}
+    )
+    public User getUserById(@ApiParam(value = "", required = true) @PathVariable("userId") UUID userId) {
+        User userById = userService.getUserById(userId);
+        channelHandler.publishUserEvent(Event.GET, userById);
+        return userById;
+    }
+
+    /**
+     * PUT /user/{userId} : Updated user
+     * This can only be done by the logged in user.
+     *
+     * @param userId (required)
+     * @param user   user Request (optional)
+     * @return This is a user info!. (status code 200)
+     * or Error Occurred (status code 400)
+     * or Error Occurred (status code 404)
+     */
+    @ApiOperation(value = "Updated user", nickname = "updateUser", notes = "This can only be done by the logged in user.", response = User.class, tags = {"user operations",})
+    @ApiResponses(value = {
+            @ApiResponse(code = 200, message = "This is a user info!.", response = User.class),
+            @ApiResponse(code = 400, message = "Error Occurred", response = GetErrorResponse.class),
+            @ApiResponse(code = 404, message = "Error Occurred", response = GetErrorResponse.class)})
+    @PutMapping(
+            value = "/user/{userId}",
+            produces = {"application/json"},
+            consumes = {"application/json"}
+    )
+    public ResponseEntity<User> updateUser(@ApiParam(value = "", required = true) @PathVariable("userId") UUID userId, @ApiParam(value = "user Request") @Valid @RequestBody(required = false) User user) {
```

Comment:
Similar to POST, the request body shouldn't be optional here.

Reply:
removed the required false option.